cmd/ui/main.go

```diff
@@ -2,49 +2,51 @@ package main
 
 import (
 	"embed"
-	"encoding/json"
+	"flag"
 	"io/fs"
 	"net/http"
+	"net/http/httputil"
+	"net/url"
 	"os"
 	"path/filepath"
 
 	"github.com/sirupsen/logrus"
 )
 
+var log = logrus.New()
+
 func main() {
-	log := logrus.New()
+	apiUrlFlag := flag.String("api-url", "http://localhost:8000", "The URL for the applications API server")
+
 	mux := http.NewServeMux()
 
 	mux.Handle("/health/", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusOK)
-	}))
-
-	mux.Handle("/api/", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		res := struct {
-			Ok bool `json:"ok"`
-		}{
-			Ok: true,
-		}
-
-		b, err := json.Marshal(res)
+		_, err := w.Write([]byte("ok"))
 
 		if err != nil {
-			log.Errorf("could not marshal: %s", err)
-			w.WriteHeader(http.StatusInternalServerError)
-			return
-		}
-
-		if _, err := w.Write(b); err != nil {
-			log.Errorf("error writing bytes: %s", err)
-			w.WriteHeader(http.StatusInternalServerError)
-			return
+			log.Error(err.Error())
 		}
 	}))
 
 	assetFS := getAssets()
 	assetHandler := http.FileServer(http.FS(assetFS))
 	redirector := createRedirector(assetFS, log)
 
+	apiUrl, err := url.Parse(*apiUrlFlag)
+
+	if err != nil {
+		log.Errorf("could not parse proxy url")
```

Comment:
You're using `Errorf` with no formatting strings, but, how about including the actual error message in the output, which might help a user diagnose the issue?

Something like `log.Errorf("could not parse proxy url: %s", err)`